CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#4737](https://github.com/bbatsov/rubocop/pull/4737): Fix ReturnInVoidContext cop when `return` is not inside a method. ([@frodsan][])
```

Comment:
Technically "not inside a context", or "in top scope". 🙂

Reply:
You can use `return` in any scope, so the description seems reasonable to me, unless I missing something @Drenmi.

Reply:
Actually I'm wrong - I tested this and you can use it only in method/lambda/proc or in the top scope. My bad.